packages/upgrade/test/__snapshots__/transforms.test.ts.snap

```diff
@@ -118,7 +118,10 @@ export function Component() {
 
   const mutation = useMutation(
     trpc.post.create.mutationOptions({
-      onSettled: () => queryClient.invalidateQueries(trpc.post.queryFilter()),
+      onSettled: () => {
+        queryClient.invalidateQueries(trpc.post.queryFilter());
+        queryClient.invalidateQueries(trpc.post.list.queryFilter());
+      },
```

Comment:
pushed up this test which is currently failing:
![CleanShot 2025-02-12 at 12 36 11](https://github.com/user-attachments/assets/cf4ad194-757a-44aa-9026-757128c3918d)

this is the output when running it on the example:

![CleanShot 2025-02-12 at 12 34 22](https://github.com/user-attachments/assets/2324a4c2-1d36-4bbf-ade5-2ca86cfc2855)

it doesn't preserve the full path

Reply:
understood, will be handled and i’ll add subsequent tests if needed as well

Reply:
yea this is very testable stuff, so easiest way I found is just to find failing cases by running the codemod on a real codebase (our examples serve as small codebases in this sense 😁) and then just copy the failing cases over to a unit test and iterate until it passes :)

I just don't have a lot of time to do it myself anymore hence the request for community help. Appreciate you taking your time to help out a ton!